x-pack/plugins/security_solution/public/detections/components/rules/select_rule_type/eql_search_icon.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="26" height="24" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Do you know if EUI will be adding the `EQL` icon to the base EUI icon set?

Reply:
I believe @marrasherrier might be working on that, but we agreed to start with an inline SVG, here.